### PR TITLE
Scenario 3 - Created the Energy Scenario

### PR DESCRIPTION
**Pull request by alod83**
examples/energy - An example of Ingestion and Processing Jobs with a Streamlit Dashboard

Why?
The objective of this example is to ingest a process in VDK two CSV datasets providing the Natural Gas price and the Climate extremes in the US. The example also builds a dashboard that shows results in streamlit.

What?
The change adds a new folder under the examples directory containing the complete example of the scenario.

How has it been tested?
The following manual tests have been run:

vdk run energy
streamlit run dashboard.py
What type of change are you making?
New Use Case

Signed-off-by: Angelica Lo Duca angelica.loduca [at] gmail.com

### CODE REVIEW

**Review by ivakoleva — COMMENTED** [review]
Did run the data job, and streamlit dashboard successfully, good work

File: examples/energy/README.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+| **1919** | 0.00              | 0.00              |
+
+## Requirements
+
+To run this example, you need:
+
+* Versatile Data Kit
+* Trino DB
+* Versatile Data Kit plugin for Trino
+* Streamlit
```
[the remark this answers is not in the document]
Comment (alod83) [reply examples/energy/README.md:74]: Yes, it is

File: examples/energy/README.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+For more details on how to install Versatile Data Kit, Trino DB, Streamlit and Versatile Data Kit plugin for Trino, please refer to [this link](https://github.com/vmware/versatile-data-kit/tree/main/examples/life-expectancy).
```
[the remark this answers is not in the document]
Comment (alod83) [reply examples/energy/README.md:76]: No, in this scenario you should configure TrinoDB manually.

**Comment by ivakoleva** [comment]
I found the use case interesting btw

File: examples/energy/dashboard.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+import pandas as pd
+import streamlit as st
+import trino
+
+# connect to server
+conn = trino.dbapi.connect(
+    host="localhost",
+    port=8080,
+    user="root",
+    catalog="mysql",
```
[the remark this answers is not in the document]
Comment (alod83) [reply examples/energy/dashboard.py:16]: It seems that Trino does not recognize the mysql catalog. When running trino in the docker you could try to configure the catalog:
`docker exec -it trino trino --catalog mysql --schema energy`

In my case, I installed everything manually, without the docker.

File: examples/energy/energy/requirements.txt [hunk 4]
```diff
@@ -0,0 +1,3 @@
+# Python jobs can specify extra library dependencies in requirements.txt file.
```
[the remark this answers is not in the document]
Comment (alod83) [reply examples/energy/energy/requirements.txt]: I have updated the requirements.txt file.

**Review by tozka — APPROVED** [review]

**Comment by tozka** [comment]
@alod83  Looks good to me and thanks for the contribution. 
If you agree and have no further changes to do, I can merge it on your behalf.

**Comment by alod83** [comment]
Yes, thanks!

**Comment by tozka** [comment]
I've merged your change. Thanks for the contributions.

**Pull request merged**